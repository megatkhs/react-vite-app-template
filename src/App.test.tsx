import { render } from "@testing-library/react";
import { App } from "@/App";

it("アプリが描画されること", async () => {
  const { container } = render(<App />);

  expect(container).toMatchSnapshot();
});
