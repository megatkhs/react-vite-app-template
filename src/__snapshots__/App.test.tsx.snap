// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`アプリが描画されること 1`] = `
<div>
  <div>
    Hello
  </div>
</div>
`;
